packages/api-request-builder/src/create-service.js

```diff
@@ -103,6 +104,12 @@ export default function createService(
           ...queryProjection,
         }
 
+      if (feature === defaultFeatures.restrictResult)
+        return {
+          ...acc,
```

Comment:
So not part of this PR but every time I see `acc` I shudder and think in this case it's just `feature`.

Reply:
But `feature` is already taken and a string :( Shouldn't it be `service` anyways, since that's the object we're putting together here?

Reply:
Yeah makes even more sense.